unpackaged/config/rd2_post_config/flexipages/NPSP_Recurring_Donation.flexipage

```diff
@@ -0,0 +1,113 @@
+        </componentInstances>
+        <name>facet-1ac173b3-bbf9-42df-b2cf-d9c3e4adb6dc</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>relatedListComponentOverride</name>
+                <value>NONE</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>rowsToDisplay</name>
+                <value>10</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>showActionBar</name>
+                <value>true</value>
+            </componentInstanceProperties>
+            <componentName>force:relatedListContainer</componentName>
+        </componentInstances>
+        <name>facet-df729d2c-0f85-4d90-ac8b-f44fa7f5078b</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>body</name>
+                <value>facet-1ac173b3-bbf9-42df-b2cf-d9c3e4adb6dc</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>title</name>
+                <value>Standard.Tab.detail</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tab</componentName>
+        </componentInstances>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>active</name>
+                <value>false</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>body</name>
+                <value>facet-df729d2c-0f85-4d90-ac8b-f44fa7f5078b</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>title</name>
+                <value>Standard.Tab.relatedLists</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tab</componentName>
+        </componentInstances>
+        <name>facet-7003451d-d97d-44c2-b74f-b56f89537a36</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>tabs</name>
+                <value>facet-7003451d-d97d-44c2-b74f-b56f89537a36</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tabset</componentName>
+        </componentInstances>
+        <name>main</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>displayNum</name>
+                <value>12</value>
+            </componentInstanceProperties>
+            <componentName>%%%NAMESPACE%%%rdScheduleVisualizer</componentName>
```

Comment:
@davidmreed I was wondering if there is a chance that CCI would be able to use the `:` separator instead of the `__` seperator for flexipages when converting `%%%NAMESPACE%%%` tokens?

Reply:
@force2b You can use the special `%%%NAMESPACE_OR_C%%%` token for those component references.

Reply:
Thank you!